app/Http/Controllers/PostController.php

```diff
@@ -18,4 +20,82 @@ public function index()
 
         return view('index')->with('posts', $posts);
     }
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('create');
+    }
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PostRequest $request)
+    {
+        $posts = new Post();
+        Post::create([
+            'name'=>$request->name,
+            'detail'=>$request->detail,
+            'author'=>$request->author
+        ]);
+        if($posts) {
```

Comment:
```suggestion
        if ($posts) {
```
chú ý convention nhé e